ethereum/datastructures/src/test/java/tech/pegasys/teku/datastructures/util/CommitteeUtilTest.java

```diff
@@ -114,7 +117,8 @@ void testIsAggregatorReturnsFalseOnARealCase() {
 
     assertThat(BLS.verify(pKey, signingRoot, selectionProof)).isTrue();
 
-    int aggregatorModulo = CommitteeUtil.getAggregatorModulo(committeeLen);
+    int aggregatorModulo =
+        specProvider.get(UInt64.ZERO).getCommitteeUtil().getAggregatorModulo(committeeLen);
```

Comment:
nit: I suspect in tests we should just create a single `Spec` or a single `CommitteeUtil` instead of having to get a spec by epoch.